docs/pages/reference/meshery-library-errors.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Meshery Lib Errors Reference
+abstract: "Meshery's library Errors"
+permalink: reference/meshkit-lib-errors
+type: Reference
+---
+## Meshkit Library Introduction

```

Comment:
Just `Introduction`? 
Unless we have a text, then I think we could just remove the section and add it later if the need arises.
Maybe a paragraph that explains that meshery artifacts are grouped into component types, and that error codes are unique within a type-name combination?